spec/mongo/client_spec.rb

```diff
@@ -444,6 +444,25 @@
         end
       end
 
+      context 'when max_pool_size and min_pool_size are both nil' do
+
+        let(:client) do
+          described_class.new(['127.0.0.1:27017'], options)
+        end
+
+        let(:options) do
+          {
+              :min_pool_size => nil,
+              :max_pool_size => nil
+          }
+        end
+
+        it 'does not set either option' do
+          expect(client.options[:max_pool_size]).to be_nil
+          expect(client.options[:min_pool_size]).to be_nil
+        end
```

Comment:
You could even change this to something like:

```ruby
it 'does not raise an exception when creating a new client'
  expect { client }.to_not raise_error
end
```

...if you like. That will make it even more clear that this test catches the (incorrect) exception being thrown. But both ways are fine, it's just a matter of "what do we want the test to verify".